docs/api/networksocket/networksocket.md

```diff
@@ -97,7 +97,7 @@ The network socket API provides a common interface for using sockets on network
 - [UDPSocket](udpsocket.html): This class provides the ability to send packets of data over UDP.
 - [TCPSocket](tcpsocket.html): This class provides the ability to send a stream of data over TCP.
 - [SocketAddress](socketaddress.html): You can use this class to represent the IP address and port pair of a unique network endpoint.
-- [CellularNonIPSocket](cellularnonipsocket.html): This class provides the ability to send and receive 3GPP Non-IP datagrams (NIDD) using Cellular IoT feature.
+- [CellularNonIPSocket](cellularnonipsocket.html): This class provides the ability to send and receive 3GPP non-IP datagrams (NIDD) using our cellular IoT feature.
```

Comment:
Cellular -> cellular is fine but "our" I dont think so. It was meant as IoT feature for cellular networks, so a cellluar IoT feature not mbed cellular IoT feature.

Reply:
Thanks for explaining. Would "...using the cellular IoT feature." be better?

Reply:
yes